test/selenium/lib/view/successful_page/accessors.js

```diff
@@ -12,7 +12,14 @@ function SuccessfulPageAccessor(driver) {
 
 SuccessfulPageAccessor.prototype = {
    get successMessageLocator() {
-        return this.driver.findElement(SELECTORS.successMessage);
+        var driver = this.driver;
+        return driver.wait(
+            webdriver.until.elementLocated(SELECTORS.successMessage))
+            .then(function (element) {
+                return driver.wait(webdriver.until.elementIsVisible(element));
+            }).then(function () {
+                return driver.findElement(SELECTORS.successMessage);
```

Comment:
It doesn't. It returns a Promise. If I don't have the findElement() it doesn't work.